server/src/routers/user.routes.ts

```diff
@@ -0,0 +1,22 @@
+import express from "express";
+
+import {
+    getAllUsers,
+    getUserById,
+    updateUserName,
```

Comment:
realistically, a user should never update their name or email (esp this one bc they login with oauth)

Reply:
Deleted